components/LogWorkout/LogWorkout.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import * as firebase from 'firebase';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
+import _, { set } from 'lodash';
 import styled from 'styled-components';
```

Comment:
Remove { set } and test this again. I think this is gonna be used instead of the actual set inside your map